test.js

```diff
@@ -607,9 +607,9 @@ test('pMapIterable - concurrency: 2', async t => {
 
 	assertInRange(t, times.get(10), {start: 0, end: 50});
 	assertInRange(t, times.get(20), {start: 0, end: 50});
-	assertInRange(t, times.get(30), {start: 200, end: 250});
-	assertInRange(t, times.get(40), {start: 300, end: 350});
-	assertInRange(t, times.get(50), {start: 300, end: 350});
+	assertInRange(t, times.get(30), {start: 195, end: 250});
+	assertInRange(t, times.get(40), {start: 295, end: 350});
+	assertInRange(t, times.get(50), {start: 295, end: 350});
```

Comment:
not sure how this is possible, but these occasionally flaked for me locally with `x97-9`ms, so I loosened the range start a bit